rails-commenting.rb

```diff
@@ -7,24 +7,35 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# A part of rails controller in MVC architecture in Ruby on Rails that coordinates an interaction
+#  b/n the user, view and model. In the above example, blog_posts_corntroller.rb is a directory of new rails controller created when we run $ rails g controller blog_post. Inside, we can define a method that allows us to define and render some type of information in our Uniform Resource Locator (url) in HTML, CSS or JS styles.
+
```

Comment:
What type of relationship is `<` showing for BlogPostsController and ApplicationController?